cmd/apps/minio_app.go

```diff
@@ -87,13 +99,20 @@ func MakeInstallMinio() *cobra.Command {
 		accessKey, _ := minio.Flags().GetString("access-key")
 		secretKey, _ := minio.Flags().GetString("secret-key")
 
+		gen, err := password.NewGenerator(&password.GeneratorInput{
+			Symbols: "+/",
+		})
+		if err != nil {
+			return err
+		}
+
 		if len(accessKey) == 0 {
 			fmt.Printf("Access Key not provided, one will be generated for you\n")
-			accessKey, err = password.Generate(20, 10, 0, false, true)
+			accessKey, err = gen.Generate(20, 10, 0, false, true)
 		}
 		if len(secretKey) == 0 {
 			fmt.Printf("Secret Key not provided, one will be generated for you\n")
-			secretKey, err = password.Generate(40, 10, 5, false, true)
+			secretKey, err = gen.Generate(40, 10, 5, false, true)
```

Comment:
Why is this called `gen` now?

Reply:
So the generator is required to set the range of allowable symbols?

Reply:
From the author of the library:
"You can customize the list of letters, symbols, numbers, etc. Create a new generator object and alter the list of symbols to be a string of the allowed symbols (e.g. "!@#$%^&*()") and it will only generate from those symbols."

```
gen, err := password.NewGenerator(&password.NewGeneratorInput{
  Symbols: "PUT YOUR LIST OF SYMBOLS HERE AS A SINGLE STRING",
})
if err != nil {
  # ...
}

gen.Generate(...)
```
https://godoc.org/github.com/sethvargo/go-password/password#NewGenerator